ingenialink/eth/net.c

```diff
@@ -391,25 +447,25 @@ static int il_eth_net_is_slave_connected(il_net_t *net, const char *ip) {
                 FD_SET(this->server, &Write);
                 FD_SET(this->server, &Err);
 
-                r = select(this->server, NULL, &Write, &Err, &Timeout);
+                #ifdef WINDOWS
+                    r = select(this->server, NULL, &Write, &Err, &Timeout);
+                #else
+                    r = select(this->server + 1, NULL, &Write, &Err, &Timeout);
+                #endif
                 if (r == 0) {
                     log_warn("Timeout during connection");
                     result = 0;
                 }
-                else {
                     if (FD_ISSET(this->server, &Write)) {
                         log_info("Connected to the Server");
                         result = 1;
                     }
                     if (FD_ISSET(this->server, &Err)) {
-                        log_warn("Fail connecting to server");
-                        result = 0;
-                    }
                 }
             }
             else {
-                int last_error = WSAGetLastError();
-                log_warn("Fail connecting to server");
+                int last_error = il_eth_get_last_socket_error();
+                log_warn("Fail connecting to server\n");
```

Comment:
Did you add the \n in the warning log?

Reply:
No, there were some merge conficts and I made some changes and did not notice the "\n". I will delete it